clients/banking/src/utils/templateTranslations.ts

```diff
@@ -163,4 +163,4 @@ export const getFeesDescription = (fees: Exclude<FeesTypeEnum, "BankingFee">) =>
       t("transaction.fees.description.processingJudicialOrAdministrativeSeizure"),
     )
     .with("UnauthorizedOverdraft", () => t("transaction.fees.description.unauthorizedOverdraft"))
-    .exhaustive();
+    .otherwise(() => "");
```

Comment:
let's rather return `undefined` here, WDYT?

Reply:
for me this is mostly the same, we just have to take care to use `isNullish` in the component using it.
Will change this

Reply:
fixed in https://github.com/swan-io/swan-partner-frontend/pull/176/commits/bea1f95d73423a054147754cf661374a4819201b